instrumentation/spring-webmvc/src/test/java/brave/spring/webmvc/WebMvcRuntimeTest.java

```diff
@@ -16,59 +16,45 @@
 import brave.http.HttpTracing;
 import brave.spring.webmvc.WebMvcRuntime.WebMvc25;
 import brave.spring.webmvc.WebMvcRuntime.WebMvc31;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.powermock.core.classloader.annotations.PowerMockIgnore;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
```

Comment:
all tests I had to use mockito for static mocking, I ported to jupiter. the next PR will switch the rest.